regnety/models/blocks.py

```diff
@@ -35,6 +35,9 @@ def __init__(self,
         self.mean = mean
         self.var = variance
 
+        self.rescale = layers.experimental.preprocessing.Rescaling(
+            scale=1./255., name="prestem_rescale"
+        )
```

Comment:
Where is the normalization layer? Or we are not using that anymore? I'm fine with either.

Reply:
No we still have it on [L44](https://github.com/AdityaKane2001/regnety/blob/3baf92359754d67e89a84782a68213a3662cb32a/regnety/models/blocks.py#L44).